src/main/resources/FormItemProfile.json

```diff
@@ -13,6 +14,7 @@
       {
         "fieldName": "Label",
         "metadataPropertyName": "label",
+        "description": "Name or main text for the question", 
```

Comment:
I think we intended to put the name/main text for the question in the (now confusingly named) default Mauro property `label`, and then this label field is the 'question label' or 'section label' from the ISO standard, i.e. the ordering instruction.

Reply:
I understand.  Changed

Reply:
The standard uses labels as order though when the 'ordered' property is true ...

Reply:
> The standard uses labels as order though when the 'ordered' property is true ...

Good point, I'd missed that bit in the standard.

So for the standard I suppose we don't technically need the separate order field. However for 'non-standard' forms I think there's a chance the separate order might be needed, so I'm inclined to keep it in the profile. As the order field is optional, if it's blank, sorting by the `label` field should be default though.

Reply:
For the standard, what happens if the intended label order is not that implied by alphabetising?  e.g. (i), (ii), (iii), (iv), (v), etc?  Or is that implementation-specific?

Reply:
ISO19763-13: "An optional Label element whose instance is a letter, number _or other ordering instruction_ associated with a Question_Element instance  given to the user completing the Form with a maximum multiplicity of one"